src/library/Uno.Material/Styles/Controls/Card.xaml

```diff
@@ -5,14 +5,83 @@
 					xmlns:controls="using:Uno.Material.Controls"
 					xmlns:toolkit="using:Uno.UI.Toolkit">
 
-	<!-- Card Variables -->
+	<!--  Card Variables  -->
```

Comment:
unnecessary change, you can change that your setting: Xaml Styler > Misc > Comment Padding > 1 (from 2)